packages/documentation/src/sidebar.js

```diff
@@ -65,6 +65,10 @@ module.exports = {
               name: 'Updating Formation',
               // href: '/getting-started/common-tasks/update-formation'
             },
+            {
+              name: 'Unit Tests',
+              href: 'getting-started/common-tasks/unit-tests',
```

Comment:
I don't think this page exists. We do have a testing common task doc that's already in the sidebar.

Reply:
I removed this and put it under https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/getting-started/common-tasks/test

This is acting as a summary - we could link to the test detail docs from this.

Reply:
So we should delete this entry in the sidebar.